package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@jridgewell/remapping",
+  "version": "0.0.3",
+  "description": "Remap sequential sourcemaps through transformations to point at the original source code",
+  "keywords": ["source", "map", "remap"],
+  "main": "dist/remapping.umd.js",
+  "module": "dist/remapping.mjs",
+  "typings": "dist/types/remapping.d.ts",
+  "files": [
+    "dist"
+  ],
+  "author": "Justin Ridgewell <jridgewell@google.com>",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jridgewell/remapping.git"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.0.0"
+  },
+  "scripts": {
+    "lint:ts": "npm run test:lint:ts -- --fix",
+    "lint:prettier": "npm run test:lint:prettier -- --write",
+    "lint": "run-s -n lint:*",
+    "prebuild": "rm -rf dist",
+    "build:ts": "tsc --module commonjs",
+    "build:rollup": "rollup -c rollup.config.js",
```

Comment:
The default location of rollup configuration is where you specified, can just use `rollup -c`